### PR TITLE
Fix/tech stack component

### PR DESCRIPTION
**Pull request by hansac91**

### CODE REVIEW

File: src/data/frontend/technologies/data.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  'Node.js',
+  'npm',
+  'Objective-C',
+  'Octopus Deploy',
+  'OpenShift',
+  'OpenStack',
+  'Oracle',
+  'Oracle Data Integrator',
+  'Oracle Service Bus',
+  'Pandas',
```
Comment (wawrzonnn) [inline src/data/frontend/technologies/data.ts:131]: where is obsługa dystrybutora lpg ?

**Review by wawrzonnn — APPROVED** [review]

File: src/components/TechStackInput/TechStackInput.module.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  align-items: center;
+  justify-content: center;
+  margin: 2px;
+  padding: 0 10px;
+  height: 36px;
+  background-color: #161a1f;
+  color: #a687ff;
+  font-size: 16px;
+  gap: 5px;
+  span {
```
Comment (wawrzonnn) [inline src/components/TechStackInput/TechStackInput.module.scss:33]: font too big

**Review by wawrzonnn — CHANGES_REQUESTED** [review]
hania

gap: 10px nigga pls

**Review by wawrzonnn — APPROVED** [review]

**Review by AniaPiWo — APPROVED** [review]

**Review by szymonkadas — APPROVED** [review]
Looks good.

**Review by mmcode9407 — APPROVED** [review]
GJ!
Please, change props on camelCase in svg element before merging :)

File: src/components/CreateProfile/WorkInformation/WorkInformation.tsx [hunk 3]
```diff
@@ -1,28 +1,53 @@
...
+  useEffect(() => {
+    const filtered = technologies.filter((tech) =>
+      tech.toLowerCase().startsWith(inputValue.toLowerCase()),
+    )
+    setFilteredSuggestions(filtered.slice(0, 8))
+  }, [inputValue])
+
+  const handleTechSelect = (tech: string) => {
+    if (!values.techStack.includes(tech)) {
+      setFieldValue('techStack', [...values.techStack, tech])
```
Comment (Sumick) [inline src/components/CreateProfile/WorkInformation/WorkInformation.tsx:37]: **nitpick:** We already have field names in some enum, let's reuse it

File: src/components/CreateProfile/WorkInformation/WorkInformation.tsx [hunk 4]
```diff
@@ -1,28 +1,53 @@
...
+    }
+    setInputValue('')
+  }
+
+  const handleTechRemove = (techToRemove: string) => {
+    if (Array.isArray(values.techStack)) {
+      setFieldValue(
+        'techStack',
```
Comment (Sumick) [inline src/components/CreateProfile/WorkInformation/WorkInformation.tsx:45]: **nitpick:** We already have field names in some enum, let's reuse it

File: src/components/CreateProfile/WorkInformation/WorkInformations.module.scss [hunk 5]
```diff
@@ -68,7 +68,7 @@
   font-size: 12px;
   color: #89939e;
   line-height: 16px;
-  margin-top: 20px;
+  // margin-top: 20px;
```
Comment (Sumick) [inline src/components/CreateProfile/WorkInformation/WorkInformations.module.scss:71]: **nitpick:** Comment to remove

**Review by Sumick — APPROVED** [review]
Good job! Im leaving 3 cleanup type comments

**Pull request merged**